### PR TITLE
Graphics API enhancements (iOS)

### PR DESCRIPTION
**Pull request by Longri**
PR for #92

### CODE REVIEW

File: vtm-ios/src/org/oscim/ios/backend/IosPaint.java [hunk 1]
```diff
@@ -337,12 +341,16 @@ public float getFontDescent() {
...
     }
 
     @Override
     public float getTextWidth(String text) {
         return measureText(text);
     }
+
+
+    public int getIosStrokeColor() {
+        return 0;
```
Comment (devemux86) [inline vtm-ios/src/org/oscim/ios/backend/IosPaint.java]: @Longri shouldn't that be:
```java
return this.strokeColorInt;
```

**Comment by Longri** [comment]
This was a remaining from debugging that method has never been used and I have removed. It was late and I was tired.

**Comment by devemux86** [comment]
So should we remove also the remaining non overridden getter methods, like `getIosStrokeCap` and `getIosLineJoin`?

**Comment by Longri** [comment]
no! we use it on IosCanvas.drawLine()

**Comment by devemux86** [comment]
:smile: 
Ok, I haven't pull the PR yet, (just asking).

**Comment by devemux86** [comment]
Thanks!

Squashed and merged via a9bb6e3788e3d735d0843674cdc20643524d7c63.

**Comment by devemux86** [comment]
@Longri do we need that initialization in [RobovmLauncher.main](https://github.com/mapsforge/vtm/blob/master/vtm-ios/src/org/oscim/ios/RobovmLauncher.java#L54) as it's called also in `createApplication` method?

**Comment by Longri** [comment]
I think no. But let me check which call we can remove. I will pull a  commit to this PR.

**Comment by devemux86** [comment]
>I will pull a commit to this PR.

This PR was merged, so please post any change in a new one.

**Comment by Longri** [comment]
oh please not for one line!
please remove the call from main()

```
public static void main(String[] argv) {
        NSAutoreleasePool pool = new NSAutoreleasePool();
        System.setProperty(org.slf4j.impl.SimpleLogger.DEFAULT_LOG_LEVEL_KEY, "TRACE");
        
        UIApplication.main(argv, null, RobovmLauncher.class);
        pool.drain();
    }
```

I have test it and its ok!

**Comment by devemux86** [comment]
Ok, I pushed the change.

**Pull request closed without merging**